lib/foods.js

```diff
@@ -1,65 +1,75 @@
 $('document').ready(function() {
+  for (var i = 0; i < localStorage.length; i++){
```

Comment:
@jdconrad89 you can think of it in a way as a session for now. we'll talk about it during class tomorrow morning 🎉